src/containers/documentation/swaggerPlugins/index.tsx

```diff
@@ -14,16 +14,16 @@ const SwaggerPlugins = (versionHandler: (newVersion: string) => void): Plugins =
   components: {
     ExtendedLayout,
     OperationTag,
-    ServersContainer: () => null,
-    authorizeBtn: () => null,
+    ServersContainer: (): null => null,
+    authorizeBtn: (): null => null,
   },
   fn: {
     curlify,
   },
   statePlugins: {
     spec: {
       wrapSelectors: {
-        allowTryItOutFor: () => () => false,
+        allowTryItOutFor: () => ((): boolean => false),
```

Comment:
Should this be:
```suggestion
        allowTryItOutFor: (): () => boolean => ((): boolean => false),
```

or is that syntax overkill? it seems less clear with the first function being typed and probably unnecessary, but just wanted to ask.

Reply:
hmm... it's a good question. what about not inlining all of this? i'm a bit reluctant to add more parens and arrows to the line bc it's starting to make me dizzy, but how about something like this:

```
const allowTryItOutSelector = (): boolean: => false;

...

    allowTryItOutFor: (): => allowTryItOutSelector,
    // OR
    allowTryItOutFor: (): (() => boolean) => allowTryItOutSelector,
```

Reply:
Agreed on not adding more parens.

Would it be out of place to add a type? 
(name of type could change at your discretion of course)
```js
type predicate = () => boolean;

...

        allowTryItOutFor: (): predicate => ((): boolean => false),
```

I'm REALLY not hung up on this, I'd be okay with your suggestion as well (`(): (() => boolean) => allowTryItOutSelector,`)

Reply:
ok! going with the `allowTryItOutSelector` route... mostly because i'm not sure i get `predicate` as the type name, and not sure what to call it? anyway, probably gonna move on hear, like you said.